apps/admin/frontend/src/components/import_err_file_modal.tsx

```diff
@@ -0,0 +1,190 @@
+  const { usbDriveStatus, electionDefinition, auth } = useContext(AppContext);
+  const importElectionResultReportingFileMutation =
+    importElectionResultsReportingFile.useMutation();
+
+  assert(electionDefinition);
+  assert(isElectionManagerAuth(auth) || isSystemAdministratorAuth(auth));
+  const [currentState, setCurrentState] = useState<ModalState>({
+    state: 'init',
+  });
+
+  async function handleImportElectionResultReportingFile(path: string) {
+    const filepath = path;
+    setCurrentState({ state: 'loading' });
+
+    const result = await importElectionResultReportingFileMutation.mutateAsync({
+      precinctId,
+      ballotStyleId,
+      votingMethod,
+      filepath,
+    });
+
+    if (result.isOk()) {
+      setCurrentState({ state: 'success' });
+    } else {
+      setCurrentState({ state: 'error', filename: filepath });
+    }
+  }
+
+  const processElectionResultReportingFileFromFilePicker: InputEventFunction =
+    async (event) => {
+      // electron adds a path field to the File object
+      assert(window.kiosk, 'No window.kiosk');
+      const input = event.currentTarget;
+      const files = Array.from(input.files || []);
+      const file = files[0] as ElectronFile;
+
+      if (!file) {
+        onClose();
+        return;
+      }
+
+      await handleImportElectionResultReportingFile(file.path);
+    };
+
+  if (currentState.state === 'error') {
+    return (
+      <Modal
+        title="Error"
+        content={
+          <P>
+            There was an error reading the contents of{' '}
+            <Font weight="bold">{currentState.filename}</Font>:{' '}
+            {currentState.errorMessage}
+          </P>
+        }
+        onOverlayClick={onClose}
+        actions={<Button onPress={onClose}>Close</Button>}
+      />
+    );
+  }
+
+  if (currentState.state === 'success') {
+    return (
+      <Modal
+        title="ERR File Added"
+        content={<P>Success</P>}
+        onOverlayClick={onClose}
+        actions={<Button onPress={onClose}>Close</Button>}
+      />
+    );
+  }
+
+  if (currentState.state === 'loading') {
+    return <Modal content={<Loading>Loading ERR File</Loading>} />;
+  }
+
+  if (
+    usbDriveStatus.status === 'no_drive' ||
+    usbDriveStatus.status === 'ejected' ||
+    usbDriveStatus.status === 'error'
+  ) {
+    return (
+      <Modal
+        title="No USB Drive Detected"
+        content={
+          <P>
+            <UsbImage src="/assets/usb-drive.svg" alt="Insert USB Image" />
+            Please insert a USB drive in order to load ERR file.
+          </P>
+        }
+        onOverlayClick={onClose}
+        actions={
+          <React.Fragment>
+            {window.kiosk && (
+              <FileInputButton
+                data-testid="manual-input"
+                onChange={processElectionResultReportingFileFromFilePicker}
+                accept=".json"
+              >
+                Select Import…
+              </FileInputButton>
+            )}
```

Comment:
I'd say only if it's very easy, since this feature doesn't need polish _and_ even with the USB inserted the file picker UX is still bad (you have to select the bookmark on the side with the low-level name of the USB drive like `usb-drive-1` or something).

Reply:
The page was already structured in a way that made disabling easy:

https://github.com/votingworks/vxsuite/pull/5344/files#diff-965242a7809fefa3e2a8ae22d9e69c4046cecf247220b6e03f9d6b772641eaddR156